dao/src/main/java/com/garret/movies/dao/entity/Movie.java

```diff
@@ -0,0 +1,37 @@
+package com.garret.movies.dao.entity;
+
+import lombok.Data;
+
+import javax.persistence.*;
+import java.sql.Date;
+import java.util.List;
+
+@Data
+@Entity
+@Table(name = "movie")
+public class Movie {
+
+    @Id
+    @Column(name = "id")
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    private Date released;
+    private Double imdbRating;
+    private Integer imdbVotes;
+    private String title;
+    private String runtime;
+    private String director;
+    @Column(length = 1024)
```

Comment:
you need this annotations when you generate DB from hibernate entities, as you disabled it and create tables by liquibase then this annotations a bit redundant